bridgetown-core/lib/bridgetown-core/concerns/site/localizable.rb

```diff
@@ -5,7 +5,7 @@ module Localizable
     # Returns the current and/or default configured locale
     # @return String
     def locale
-      if @locale
+      if defined?(@locale)
```

Comment:
let's just memoize this via a `@locale ||= begin` block

Reply:
Ah nice, I didn't know you could use `begin` / `end` in this way!